docker-compose.yml

```diff
@@ -4,9 +4,14 @@ services:
   db:
     image: postgres:10
     ports:
-      - "5432:5432"
+      # We changed the default port so people having already a postgres instance running on port 5432 won't have a port conflict
+      - "5433:5432"
```

Comment:
Simple trick to make the life of people who has already a postgres instance running (like me) easier during the install.